docs/_documentation/plugins/share.md

```diff
@@ -21,5 +21,5 @@ On iOS, currently only sharing by linked Twitter account is supported. There is
 
 A sample using the Share plugin is:
 
-- Conference - see https://github.com/slodge/MvvmCross-Tutorials/tree/master/Sample%20-%20CirriousConference
+- Conference - see hhttps://github.com/slodge/MvvmCrossConference
```

Comment:
This should link to https://github.com/MvvmCross/MvvmCross-Samples/tree/master/OldSamples/CirriousConference